docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:18.04 as built
```

Comment:
Why ubuntu for something so lightweight? And why not use a standard golang base image, with say, alpine linux as OS?

Reply:
resolved. using golang:alpine as base now